src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -13,21 +13,180 @@
 #
 #
 
-.rs.addJsonRpcHandler("preview_data_import", function(path, maxRows, maxCols = 100, maxFactors = 64)
+.rs.addFunction("assemble_data_import_name", function(dataImportOptions)
+{
+  dataName <- dataImportOptions$dataName
```

Comment:
Yes, for sure. However, I noticed 2-spacing in, for instance, `SessionDataViewer.R`. Therefore, please chime in if 3-space is not our preference, thanks!